app/views/idv/doc_auth/_combined_upload.html.erb

```diff
@@ -0,0 +1,76 @@
+<% title t('titles.doc_auth.upload') %>
+
+<%= render 'idv/doc_auth/error_messages', flow_session: flow_session %>
+
+<%= render PageHeadingComponent.new do %>
+    <%= t('doc_auth.headings.combined_upload') %>
+<% end %>
+
+<p>
+    <%= t('doc_auth.info.combined_upload') %>
+</p>
+
+<hr class="margin-y-4" />
+
+<div class="grid-row">
+    <div class="grid-col-12 tablet:grid-col-3">
+        <%= image_tag(
+              asset_url('idv/phone-icon.svg'),
+              alt: t('image_description.camera_mobile_phone'),
+              width: 100,
+              height: 100,
+            ) %>
+    </div>
+    <div class="grid-col-12 tablet:grid-col-9">
+        <div class="usa-tag bg-primary-dark margin-top-1">
+            <%= t('doc_auth.info.tag') %>
+        </div>
+        <h2 class="margin-y-105">
+            <%= t('doc_auth.headings.combined_upload_from_phone') %>
+        </h2>
+        <%= t('doc_auth.info.combined_upload_from_phone') %>
+        <%= simple_form_for(
+              idv_phone_form,
+              as: :doc_auth,
+              url: url_for(type: :mobile),
+              method: 'PUT',
+              html: { autocomplete: 'off' },
+            ) do |f| %>
+            <%= render PhoneInputComponent.new(
+                  form: f,
+                  required: false,
+                  delivery_methods: [:sms],
+                  class: 'margin-bottom-4',
+                ) %>
+            <%= f.submit t('forms.buttons.send_link'), wide: false %>
+        <% end %>
+    </div>
+</div>
+
+<hr class="margin-y-4" />
+<div class="grid-row">
+    <div class="grid-col-12 tablet:grid-col-3">
+        <%= image_tag(
+              asset_url('idv/laptop-icon.svg'),
+              alt: t('image_description.laptop'),
+              width: 100,
+              height: 100,
+            ) %>
+    </div>
+    <div class="grid-col-12 tablet:grid-col-9">
+        <h2 class="margin-y-105">
+            <%= t('doc_auth.headings.combined_upload_from_computer') %>
+        </h2>
+        <%= t('doc_auth.info.combined_upload_from_computer') %>&nbsp;
+        <%= simple_form_for(
+              :doc_auth,
+              url: url_for(type: :desktop),
+              method: 'PUT',
+              class: 'margin-bottom-4',
+            ) do |f| %>
+            <%= f.submit t('forms.buttons.upload_photos'), class: 'usa-button--outline', wide: false %>
```

Comment:
You can assign `outline` as a keyword argument (`f.submit` passes through to `SubmitButtonComponent`, which extends `ButtonComponent`)

```suggestion
            <%= f.submit t('forms.buttons.upload_photos'), outline: true, wide: false %>
```